controls/radribbonview/gettingstarted.md

```diff
@@ -27,38 +27,40 @@ This tutorial will walk you through the creation of a sample application that co
 
 ## Assembly References
 
-In order to use the __RadRibbonView__ control, you will need to add references to the following assemblies:
+In order to use the RadRibbonView control, you will need to add references to the following assemblies:
+
 * __Telerik.Windows.Controls__
 * __Telerik.Windows.Controls.Input__
 * __Telerik.Windows.Controls.Navigation__
 * __Telerik.Windows.Controls.RibbonView__
 * __Telerik.Windows.Data__
 
-You can find the required assemblies for each control from the suite in the {% if site.site_name == 'Silverlight' %}[Controls Dependencies]({%slug installation-installing-controls-dependencies%}){% endif %}{% if site.site_name == 'WPF' %}[Controls Dependencies]({%slug installation-installing-controls-dependencies-wpf%}){% endif %} help article.
-
 ## Defining a RadRibbonView
 
-You can add a __RadRibbonView__ manually in XAML as demonstrated in __Example 1__.
+You can add a RadRibbonView manually in XAML as demonstrated in the following example:
 
-#### __[XAML] Example 1: Adding a RadRibbonView in XAML__
+#### __[XAML] Adding a RadRibbonView in XAML__
 {{region radribbonview-gettingstarted-0}}
 	<telerik:RadRibbonView />
 {{endregion}}
 	
-At this state, the ribbonview is empty - there are not tabs, backstage or any other ribbon components. You can learn about the controls that the __RadRibbonView__ can contain from the [Features]({%slug radribbonview-ribbon-tab%}) section of the documentation.
+At this state, the ribbonview is empty - there are not tabs, backstage or any other ribbon components. You can learn about the controls that the RadRibbonView can contain from the [Features]({%slug radribbonview-ribbon-tab%}) section of the documentation.
 
-#### __Figure 1: Empty RadRibbonView__
-![{{ site.framework_name }} RadRibbonView Empty RadRibbonView](images/ribbonview-getting-started-01.png)
+__Empty RadRibbonView__
+
+![WPF RadRibbonView Empty RadRibbonView](images/ribbonview-getting-started-01.png)
 
 ## Adding Ribbon Tabs
 
-The __RadRibbonView__ helps users to quickly find the tools and options they need in order to complete a task. Tools and options are organized in logical groups that are collected together under specific tabs. The ribbon tabs allows you to categorize the commands to be displayed to the users. The class that represents the ribbon tab is the __RadRibbonTab__.
+The RadRibbonView helps users to quickly find the tools and options they need in order to complete a task. Tools and options are organized in logical groups that are collected together under specific tabs. The ribbon tabs allows you to categorize the commands to be displayed to the users. The class that represents the ribbon tab is the `RadRibbonTab`.
+
+>tip For more information about the RadRibbonTab, check out the [Ribbon Tab]({%slug radribbonview-ribbon-tab%}) help article. Also, keep in mind that you can use contextual tabs. You can read more about this in the [Contextual Tabs]({%slug radribbonview-contextual-tabs%}) topic.
```

Comment:
Consider moving the Ribbon Tab link from the "tip" to the RadRibbonTab word in the previous sentence. For example: 

"The class that represents the ribbon tab is the [RadRibbonTab](link)".

or

"The class that represents the ribbon tab is the `RadRibbonTab`. Read more about the tabs in the [Ribbon Tab] article."

This will leave only the contextual tabs information in the "tip".